resources/configurations/examples/change_domain.yaml

```diff
@@ -32,8 +32,8 @@ SimpleParticipant_domain1:                                                  # 6
 ##################################
 # CONFIGURATION DESCRIPTION
 
-# This configuration example configures a DDS Router to listen to every message published in two different domains
-# and transmit those messages trough the other domain
+# With this example, a DDS Router listens to every message published in two different domains and transmit
```

Comment:
In general:
Be careful, these are **Configuration examples**, not **Router examples**.
The explanation of each example must be done in the documentation and not in these files.

Reply:
:+1: